script.js

```diff
@@ -0,0 +1,37 @@
+// Google Maps:  Still need to research to find query URL, etc.
+
+// API Key:
+
+var gKey = "AIzaSyDQOySmk8taGDt9pVaSXmNHpO0jjMnQkJ8";
+
+function getCounty(lat, lon) {
+  var queryLocation = lat + ", " + lon;
+  var url =
+    "https://maps.googleapis.com/maps/api/geocode/json?latlng=" +
+    queryLocation +
+    "&sensor=false&key=" +
+    gKey;
+  console.log(url);
```

Comment:
Remove all console.log() or comment out before PR

Reply:
I would like to keep in console.log() checks so we can use them as we piece the different functions together and then remove them when those when we have a functioning mvp. I will make them all commented out though.